docs/src/index.tsx

```diff
@@ -138,7 +144,7 @@ const App = () => {
             <VuiFlexItem grow={false} shrink={false}>
               <VuiTitle size="xs">
                 <h1>
-                  <strong>Vectara React-Chatbot</strong>
+                  <strong></strong>
```

Comment:
Is this supposed to be empty now?

Reply:
https://github.com/vectara/react-chatbot/pull/9/commits/d439e717267622629e3ea5c4983c52e9dfa59d51